extensions/notebook/src/book/bookModel.ts

```diff
@@ -182,95 +178,83 @@ export class BookModel {
 	public async getSections(tableOfContents: IJupyterBookToc, sections: JupyterBookSection[], root: string, book: BookTreeItemFormat): Promise<BookTreeItem[]> {
```

Comment:
Sections were a bit confusing to me, I had to go to the TOC and also look over this [documentation](https://jupyterbook.org/customize/toc.html#how-headers-and-sections-map-onto-to-book-structure) to get a better understanding. 
It would be great to add some comments similar to js doc /** to explain these methods :)

Reply:
Sure, just to clarify getSections is called everytime we expand a node in the BookTreeView to show the notebooks and sections under that node.

Reply:
Sure, I would say it would be good to have a high-level understanding as well. As an analogy I used for sections was that it's basically a notebook that acts as a chapter in a book. 
Seems like there are a lot of book contexts that I would have to refresh myself on. 
Book level -> sections (notebooks) -> inidvidual notebooks